ledfx/virtuals.py

```diff
@@ -990,6 +1024,48 @@ def config(self, new_config):
             VirtualConfigUpdateEvent(self.id, self._config)
         )
 
+        if reactivate_effect:
+            self._reactivate_effect()
+
+    @property
```

Comment:
Should this be @cached_property and handled in invalidate_cached_props

Just to follow existing pattern for performance

Reply:
I went for simplicity here since the calculation overhead is probably negligible

Reply:
Not being the originator of the code, its difficult to have a strong opinion here, but seeing as they went to the trouble of using the cached property and it is in the root of a LOT of execution, I would recommend following the pattern. I know its cargo cult, but leaving it half and half is the worst situation.

Reply:
c623f79b3f64cd777c18eda51a1cf1c85c5e7851